bindings/nodejs/lib/AccountManager.ts

```diff
@@ -375,4 +375,14 @@ export class AccountManager {
             payload: { mnemonic },
         });
     }
+
+    /**
+     * Update the authentication for the provided node.
+     */
+    async updateNodeAuth(url: string, auth?: Auth): Promise<void> {
```

Comment:
Why is it optional? So we can remove auth?

Reply:
Yes